coins/btdx.json

```diff
@@ -66,8 +68,8 @@
                 }
             }
         },
-        "hash_genesis_block": "00008412c3a4bbf7133c9cfbb8f9041fdd39bc98d1e1527612d10d95ed9b06d1",
-        "message_magic": "\u0018Diamond Signed Message:\n",
+        "hash_genesis_block": "000002d56463941c20eae5cb474cc805b646515d18bc7dc222a0885b206eadb0",
```

Comment:
Why is hash_genesis_block and message_magic changing?     Do we have a symbol collision for btdx?

Reply:
See https://github.com/LIMXTEC/Bitcloud/blob/3287f756c21dd27d12a638cccbd4c656ae8acccf/src/chainparams.cpp#L191

Reply:
okay, so that line hasn't changed since a year ago.  [evidence](https://github.com/LIMXTEC/Bitcloud/blame/3287f756c21dd27d12a638cccbd4c656ae8acccf/src/chainparams.cpp#L191).  I don't understand yet why it was previously different in btdx.json.  I need to investigate a bit more.

regarding message_magic, it still contains `Diamond`.  defined here: https://github.com/LIMXTEC/Bitcloud/blob/3287f756c21dd27d12a638cccbd4c656ae8acccf/src/main.cpp#L99.  So where are you getting the `BitCloud` version from?

Reply:
ok, I see.  the `0x00008412c3a...` genesis block value is in a long comment.  The parser isn't that smart, so it just used the first one.  Your value is correct. 

message_magic question remains.

Reply:
sorry, my fault. fixed